test/python/result/test_quasi.py

```diff
@@ -0,0 +1,27 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test conversion to probability distribution"""
+from math import sqrt
+from qiskit.result import QuasiDistribution
+
+
+def test_known_quasi_conversion():
```

Comment:
This actually doesn't run as is, this will need to be in a test class so unittest discovery will pick it up.

Reply:
Ahh yeah sorry, it came from a repo using pytest .

Reply:
done